adapter/pkg/synchronizer/types.go

```diff
@@ -63,3 +63,15 @@ type GatewayLabel struct {
 	Name  string `json:"name"`
 	Vhost string `json:"vhost"`
 }
+
+// APIConfigs represents env properties belongs to the API
```

Comment:
these structs are common to both synced and apict pushed APIs. so should be placed out of `synchronizer` pkg. `ProjectAPI` struct is in `oasparser` pkg. thats where this should be defined IMO

Reply:
This struct is used for the env_properties.json which is coming from APIM side. not supported for apictl at the moment. Hence keeping it in the synchronizer.